ompi/mca/osc/rdma/osc_rdma_component.c

```diff
@@ -1235,6 +1248,9 @@ static int ompi_osc_rdma_component_select (struct ompi_win_t *win, void **base,
 
     /* fill in our part */
     ret = allocate_state_shared (module, base, size);
+
+    /* notify all others if something went wrong */
+    ret = synchronize_errorcode(ret, module->comm);
```

Comment:
This I see as the most likely scenario. Run into problems on one/some nodes but not all nodes.